code/modules/antagonists/eldritch_cult/eldritch_knowledge.dm

```diff
@@ -1,6 +1,6 @@
 
 /**
- * #Eldritch Knwoledge
+ * #Eldritch Knowledge
```

Comment:
I figured that a single word in a single comment was not worthy of the grammar and formatting tag, but I still ought to fix it.